### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by darxriggs**

### CODE REVIEW

File: src/test/java/hudson/security/PAMSecurityRealmTest.java [hunk 1]
```diff
@@ -12,9 +16,14 @@
...
-    public void testLoadUsers() {
-        if (Functions.isWindows())  return; // skip on Windows
+public class PAMSecurityRealmTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    @Test
+    public void loadUsers() {
+        if (Functions.isWindows()) return; // skip on Windows
```
Comment (jvz) [inline src/test/java/hudson/security/PAMSecurityRealmTest.java]: You can use `assumeTrue(Functions.isWindows());`

Comment (darxriggs) [reply src/test/java/hudson/security/PAMSecurityRealmTest.java]: I guess you meant `assumeFalse(...)`. I just applied this change.

**Comment by darxriggs** [comment]
There has been a hiccup on ci.jenkins.io. Closing & reopening this pull request to retrigger a build.

**Comment by darxriggs** [comment]
@jvz All tests passed. But the last cleanup step on Windows seems to have an issue.

**Comment by darxriggs** [comment]
@jvz how do we go on with this? Can you merge it (all tests passed) even if the build failed?

**Comment by jvz** [comment]
We can merge now, no worries. Thanks for the contribution!

**Pull request merged**